include/ygm/detail/cereal_boost_json.hpp

```diff
@@ -5,6 +5,9 @@
 
 #pragma once
 
+// This prevents the compilation of the header if boost is not linked.
+#if __has_include(<boost/json/src.hpp>)
```

Comment:
Instead of guarding the full file, can you just put a fail "#error Boost >= 1.75 is required for Boost.JSON".

Reply:
Sure, that is an easy change!

Reply:
it is done

Reply:
Thanks.   I'm impressed how well these in-line comments work...